packages/thirdweb/src/extensions/erc1155/write/sigMint.ts

```diff
@@ -135,7 +135,14 @@ export async function generateMintSignature(
       return "";
     })(),
     // uid computation
-    mintRequest.uid || (await randomBytesHex()),
+    ((): Hex => {
+      if (mintRequest.uid) {
+        return isHex(mintRequest.uid)
+          ? mintRequest.uid
+          : stringToHex(mintRequest.uid, { size: 32 });
```

Comment:
Just confirming this produces a 32 byte hex, not a 32 char hex

Reply:
Correct